packages/tasit-action/src/testHelpers/helpers.js

```diff
@@ -95,6 +97,9 @@ const mineOneBlock = async provider => {
 };
 
 const mineBlocks = async (provider, n) => {
+  // Do nothing if provider isn't a JSON-RPC
```

Comment:
What's an example of such a provider?

Reply:
Infura uses RPC calls over HTTP instead of the RPC protocol directly.

Reply:
Ohhh, interesting. So maybe that's worth mentioning specifically in the comment. I assume it's the same for Etherscan and Alchemy too.

Reply:
```suggestion
  // Do nothing if provider isn't a JSON-RPC
  // (Infura uses RPC calls over HTTP as opposed to JSON-RPC directly)
```